addons/tmf_loadouts/loadouts/US_USMC_1985.hpp

```diff
@@ -0,0 +1,493 @@
+	scope[] = {};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_9x19_MP5"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_2("CUP_1Rnd_SmokeRed_M203")
+	};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {"usm_vest_pasgtdes_lbe_rmp_m"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"usm_bdu_8point_des_m"};
+	linkedItems[] = {"ItemGPS"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc119"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"CUP_lmg_M249_E2"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	vest[] = {"usm_vest_pasgtdes_lbe_mg_m"};
+	backpack[] = {"CUP_B_AlicePack_Khaki"};
+	magazines[] = {
+		LIST_3("CUP_200Rnd_TE4_Red_Tracer_556x45_M249"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"usm_pack_200rnd_556_bandoliers"};
+	Items[] += {"ACE_EntrenchingTool"};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : r {
+	displayName = "Rifleman (AT)";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_Mk153Mod0"};
+	magazines[] += {"CUP_SMAW_HEAA_M"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"hlc_rifle_M21"};
+	scope[] = {"hlc_optic_artel_m14"};
+	magazines[] = {
+		LIST_9("ACE_10Rnd_762x51_M118LR_Mag"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_M60"};
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	magazines[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class mmgac : aar {
+	displayName = "MMG Ammo Carrier";
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_M2_Gun_Bag"};
+};
+
+class hmgac : car {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_M2_MiniTripod_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_M2_Tripod_Bag"};
+	linkedItems[] += {"Binocular"};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"CUP_launch_M47"};
```

Comment:
Use ACE launcher instead

Reply:
There's an ACE M47 Dragon? I can't seem to find it in the arsenal, can you provide me the location/item name?

Reply:
`ace_dragon_super` and the `ace_dragon_sight` CLU. ofc these are the Super Dragon, but it is a lot more authentic in the way you use it compared to the old CUP dragon.